virttest/ceph.py

```diff
@@ -109,6 +117,12 @@ def rbd_image_exist(ceph_monitor, rbd_pool_name, rbd_image_name,
     opts = m_opt + ' ' + c_opt
     keyring = '--keyring %s' % keyfile if keyfile else ''
     cmd = cmd.format(opts=opts, pool=rbd_pool_name, keyring=keyring)
+    if rbd_image_namespace is not None:
```

Comment:
we might not need to update cmd when rbd_image_namespace is set, e.g.

cmd = "rbd {opts} ls {pool}{namespace} {keyring}"
namespace = "/%s" % namespace if namespace else ""

Reply:
Thanks for your advice, indeed, no need to add if branch of namespace. Fixed now.